bin/uglifyjs

```diff
@@ -113,6 +113,9 @@ function process_option(name, no_value) {
                 "  --warn                                   Print warning messages.",
                 "  --webkit                                 Support non-standard Safari/Webkit.",
                 "  --wrap <name>                            Embed everything as a function with “exports” corresponding to “name” globally.",
+                "",
+                "(use for debug)",
+                "  --in-situ                                Process and write back a list of files.",
```

Comment:
> I don't think `--in-situ` should be documented - at least for now. It's for debugging and it may change.

Since we did document `--reduce-test`, I've added this debug disclaimer so if somebody uses it they get to keep the pieces 👻

Reply:
I did see that. But because it's destructive to the original source I wouldn't document it or encourage its use just yet. They'd lose all their comments, at the very least.

If you do decide to keep the documentation, I'd suggest `internal debug use only` or something like that and to change `Process and write back a list of files` to `Warning: replaces original source files with minified output`.

Reply:
You are the man with the words 😉 

Help text updated in https://github.com/mishoo/UglifyJS/pull/4758/commits/66314431d683780d50f92d2e7d849cebff9abf9d